site/includes/footer.php

```diff
@@ -1,6 +1,34 @@
-        <footer class="bg-dark">
-            <p class="h2 mb-4">Open Game Data</p>
-            <img class="logo" src="assets/images/logos/FieldDay-logo.png" alt="Field Day">
+        <footer>
```

Comment:
The following padding classes should improve the available space at the different breakpoints: `p-2 py-md-5 px-lg-5`